### DISCUSSION TITLE
DateTimePicker / Date Input

### DISCUSSION THREAD

**Discussion by ckaldemeyer** [flet-dev/flet#553] [Q&A] [answered]
Hi everybody,

thanks for creating `flet` which I found really promising in my first tests.

For the start, I am considering it as a more versatile alternative to create my `streamlit` and `plotly_dash` apps with the opportunity to go "native" on different devices if needed.

I found almost everything I need except for one thing: the functionality to pick date and time in a convenient way e. g. through DateTimePickers such as for [streamlit](https://docs.streamlit.io/library/api-reference/widgets/st.date_input) or [dash](https://dash.plotly.com/dash-core-components/datepickersingle).

For flutter, I found the respective [functionality](https://api.flutter.dev/flutter/material/showDatePicker.html) as well. Is this already available in `flet` or is there a way to add it manually?

Best wishes,
Cord

**Comment by ndonkoHenri** [comment, marked as answer]
https://github.com/flet-dev/flet/issues/86

Subscribe to this issue.

**Reply by ckaldemeyer** [reply]
Thanks!
